src/server/index.jsx

```diff
@@ -260,7 +260,11 @@ server
           routes,
           urlPath,
         );
-        const data = await route.getInitialData(url);
+        const data = await route.getInitialData({
```

Comment:
This has been updated across the board in https://github.com/bbc/simorgh/pull/6109 - we can pull those changes into this branch if we want to merge the radio schedules work first.

Reply:
This is now in https://github.com/bbc/simorgh/pull/6171/files